packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -193,14 +226,16 @@ export class TreeViewConfiguration<TSchema extends ImplicitFieldSchema = Implici
 
 			node: markSchemaMostDerived,
 			allowedTypes(types): void {
-				if (config.ensureUnambiguous) {
+				if (config.preventAmbiguity) {
 					checkUnion(types, ambiguityErrors);
 				}
 			},
 		});
 
 		if (ambiguityErrors.length !== 0) {
-			throw new UsageError(`Ambagious Schema Found:\n${ambiguityErrors.join("\n")}`);
+			// Duplicate errors are common since when two types conflict, both orders error:
+			const deduplicated = new Set(ambiguityErrors);
+			throw new UsageError(`Ambagious Schema Found:\n${[...deduplicated].join("\n")}`);
```

Comment:
```suggestion
			throw new UsageError(`Ambigious schema found:\n${[...deduplicated].join("\n")}`);
```

Reply:
I see your `Ambigious` and raise you an `Ambiguous` 😄 .

Reply:
Ambugious